airflow-core/src/airflow/config_templates/config.yml

```diff
@@ -2588,6 +2509,17 @@ dag_processor:
       type: integer
       example: ~
       default: "30"
+    health_check_threshold:
```

Comment:
Should this be called `dag_processor_health_check_threshold` to match existing pattern?